.deepsource.toml

```diff
@@ -0,0 +1,42 @@
+version = 1
+
+exclude_patterns = ["**/node_modules/", "**/dist/", "**/build/", "**/target/", "**/pkg/", "**/.turbo/"]
+test_patterns    = ["**/*.test.ts", "**/*.spec.ts"]
+
+[[analyzers]]
+name = "javascript"
+
+[analyzers.meta]
+environment = [
+    "vitest",
+    "nodejs",
+    "browser",
+]
+plugins = ["react"]
```

Comment:
I quite dislike TOML for nested maps. I don't mind keeping it, though.

Reply:
fixed in v12